llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -5690,6 +5690,38 @@ Value *SPIRVToLLVM::transSPIRVImageAtomicOpFromInst(SPIRVInstruction *bi, BasicB
   return result;
 }
 
+// =============================================================================
+// Helper function for handling converting sampler select ladder
+Value *SPIRVToLLVM::ConvertingSamplerSelectLadderHelper(Value *result, Value *convertingSamplerIdx,
+                                                        std::function<Value *(Value *)> alternativeFunc) {
```

Comment:
The comment in the function body is great, but this function doesn't have a descriptive documentation on top in neither the header nor the .cpp file. Can you follow the usual documentation style for functions, and maybe show an ascii diagram or pseudocode to get an idea of what code pattern this function creates?

Reply:
I've followed the usual documentation style for this function. I'm still waiting if there's some comments from others. 

Regarding to more detailed introduction, i think @trenouf has the whole page of the role that this function plays, the idea (and this function) is from him, maybe a follow up commit could be possible when he has time.